website/pages/blog/offline-graphql-the-easy-parts.mdx

```diff
@@ -0,0 +1,176 @@
+---
+title: Offline GraphQL - The easy parts
```

Comment:
I would mention Flutter in the title, otherwise people might think it can be used on other ecosystems

Reply:
maybe something like
"Brick - a new, offline first GraphQL Client based on SQLite"?
Or something like that?